packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -74,7 +74,9 @@ if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryRespon
 				try {
           const hasTitle = decodedAsString.indexOf("<title>") > -1;
           if (hasTitle) {
-					  var title = decodedAsString.split("<title>")[1].split("</title>")[0];
+            const firstTitle = decodedAsString.split("<title>")[1];
+            if (!firstTitle) return;
```

Comment:
Returning here is a big deal, the previous logic was just ignoring the title if not present.

Also, I'd suggest removing the `hasTitle` and using `firstTitle` there as the first check instead - no need for both.